app/forms/csv_upload_form.rb

```diff
@@ -17,9 +17,10 @@ def save(file)
     @file = file
     return false unless file && valid?
 
-    ActiveRecord::Base.transaction { keywords.each(&:save) }
+    saved_keywords = []
+    ActiveRecord::Base.transaction { saved_keywords += keywords.each(&:save) }
 
-    errors.empty?
+    errors.empty? ? saved_keywords : false
```

Comment:
If you agree with the "Active Record callback" usggestion, we do not need this change.

Also please remember the part about returning `boolean` or `object / nil`, but avoiding to return an object OR a boolean in the same function 🥴

Reply:
I migrated to Action Record Callback. Thank you for the suggestion. 🙏